meta-resin-common/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -17,3 +17,8 @@ do_install_append() {
 
 # Network configuration is managed by connman
 PACKAGECONFIG_remove = "resolved networkd"
+
+# Add missing users/groups defined in /usr/lib/sysusers.d/*
+# In this time we avoid creating these at first boot
+USERADD_PARAM_${PN} += "; --system systemd-bus-proxy; --system --system -d / -M --shell /bin/nologin -u 65534 nobody;"
```

Comment:
Is the second user adding the --system flag twice an accident?
Also, can we stick to either the short or the long form of parameters? The mix of both is a bit strange